ginipaylib/src/androidTest/java/net/gini/android/DocumentTaskManagerTest.java

```diff
@@ -441,86 +442,14 @@ public void testDeleteDocumentDeletesParentsFirst() throws Exception {
                 .deleteDocument(eq(document.getId()), eq(mSession));
     }
 
-    @Test
```

Comment:
Could you give a bit of details why did we remove these tests?

Reply:
`DocumentUploadBuilder` decided what method to create for uploading a document based on what was provided (Bitmap or bytes). Since I removed the support for bitmaps `DocumentUploadBuilder` wasn't needed anymore so I deleted it.

The tests only checked that `DocumentUploadBuilder` called correctly the method for uploading a document, but it doesn't exist anymore.